packages/cssnano/src/__tests__/postcss-calc.js

```diff
@@ -1,5 +1,7 @@
 import processCss from './_processCss';
 
+jest.setTimeout(30000);
```

Comment:
Avoid it and use `setupFilesAfterEnv` https://jestjs.io/docs/en/jest-object#jestsettimeouttimeout

Reply:
ok cool, I will try this 👍